helm-charts/helm3/strimzi-kafka-operator/crds/040-Crd-kafka.yaml

```diff
@@ -848,6 +881,9 @@ spec:
                                       - certificate
                                       - secretName
                                   description: Trusted certificates for TLS connection to the OAuth server.
+                                tokenEndpointUri:
```

Comment:
Aside: I do see the `Uri` in this name and feel bad about the inconsistency with `enableECDSA`. In hindsight it should have been `enableEcdsa`. Maybe that's something we should change for `v1beta2`, WDYT @scholzj @ppatierno @alesj ?

Reply:
If we replace `keycloak-core` with `nimbus-jose-jwt` we could possibly remove the `enableECDSA` flag altogether because I think it would be enabled automatically without the need to install Bouncycastle crypto providers.